web/onos-gui/src/app/onos-topo/devices-list/devices-list.component.css

```diff
@@ -35,18 +48,21 @@
     color: #3c3a3a;
 }
 
-#ov-devices div.summary-list .table-body {
-    overflow: scroll;
+#ov-devices div.summary-list .table-sticky tbody {
+    height: 250px;
```

Comment:
I finally figured out how to do this - set the height to be 
100vh (the browser height ) minus 162px (the hight of fixed elements)
```
    height: calc(100vh - 162px);
```

Reply:
changed